src/Pages/GlobalConfigurations/Authorization/shared/components/AppPermissions/types.ts

```diff
@@ -0,0 +1,43 @@
+import { ServerError } from '@devtron-labs/devtron-fe-common-lib'
+import { JobList } from '../../../../../../components/Jobs/Types'
+import { ACCESS_TYPE_MAP } from '../../../../../../config'
+import { DirectPermissionsRoleFilter } from '../../../types'
+
+type AppsList = Map<number, { loading: boolean; result: { id: number; name: string }[]; error: ServerError }>
+type JobsList = Map<number, { loading: boolean; result: JobList['result']['jobContainers']; error: ServerError }>
+
+export interface AppPermissionsDetailType {
+    accessType: ACCESS_TYPE_MAP.DEVTRON_APPS | ACCESS_TYPE_MAP.HELM_APPS | ACCESS_TYPE_MAP.JOBS
```

Comment:
Can't we directly set ACCESS_TYPE_MAP directly rather than specifying it?

Reply:
No, the app permission detail is only for devtron apps, helm apps and jobs. For K8s and Charts, we have separate components as there is no reuability among them